test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatCollection.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    [CollectionDefinition("NuGet XPlat Test Collection")]
```

Comment:
Those guidelines do not require to use long names or to follow any naming convention:
> Create the collection definition class, decorating it with the [CollectionDefinition] attribute, giving it a unique name that will identify the test collection.

Anyway, just saying this'd make one's life easier.